fenix/app/src/main/java/org/mozilla/fenix/compose/tabstray/TabGridItem.kt

```diff
@@ -146,13 +149,17 @@ fun TabGridItem(
                             .fillMaxWidth()
                             .wrapContentHeight(),
                     ) {
-                        Favicon(
-                            url = tab.content.url,
-                            size = 16.dp,
-                            modifier = Modifier
-                                .align(Alignment.CenterVertically)
-                                .padding(start = 8.dp),
-                        )
+                        Spacer(modifier = Modifier.width(8.dp))
+
+                        tab.content.icon?.let {
+                            Image(
+                                bitmap = it.asImageBitmap(),
+                                contentDescription = null,
+                                modifier = Modifier
+                                    .align(Alignment.CenterVertically)
+                                    .size(16.dp),
+                            )
+                        }
```

Comment:
Does this mean that we won't load a favicon for the corner if it isn't already in memory?

Reply:
That's correct, and this is how the [XML implementation does it](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/tabstray/browser/AbstractBrowserTabViewHolder.kt#122).

Reply:
Oh okay cool. Thanks